3ds Max/Max2Babylon/Forms/MultiExportForm.cs

```diff
@@ -216,6 +250,61 @@ private void ExportItemGridView_CellDoubleClick(object sender, DataGridViewCellE
                     ExportItemGridView.EndEdit();
                 }
             }
+
+
+            // double-clicked layers column cell, select some layers!
+            if(e.ColumnIndex == ColumnLayers.Index)
+            {
+                if (sceneExplorer==null )
+                {
+                    sceneExplorer = new SceneExplorerDialog("Layer to Export");
+                    sceneExplorer.ExplorerControl.EditingEnabled = false;
+                    sceneExplorer.Closed += SceneExplorerOnClosed;
```

Comment:
This doesn't seem to work very well. There's no UI to confirm your layer selection, and closing the scene explorer doesn't appear to select the layers as well. Also, selecting the objects instead of the layers that you want to export doesn't make for a very good experience as well.

![image](https://user-images.githubusercontent.com/3952729/63986266-092fc780-ca88-11e9-89cd-d17af3837967.png)


@PatrickRyanMS currently this proposes that double clicking a Layer cell opens the scene explorer to select objects, and the layers of those selected objects should be added to the multi-export item.